build-common/NHibernate.props

```diff
@@ -13,11 +13,13 @@
     <FileVersion Condition="'$(VersionSuffix)' != '' AND '$(BuildNumber)' != ''">$(VersionPrefix).$(BuildNumber)</FileVersion>
     <FileVersion Condition="'$(FileVersion)' == ''">$(VersionPrefix).0</FileVersion>
 
-    <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net461;netcoreapp2.0</NhAppTargetFrameworks>
+    <NhAppTargetFrameworks Condition ="$(NhAppTargetFrameworks) == ''">net461;netcoreapp2.0;netcoreapp2.1</NhAppTargetFrameworks>
```

Comment:
To work around the async generation issue, I target both net core 20 and 2.1.
The test explorer in Visual Studio 2022 then show tests for three targets, net461, netcoreapp2.0 and 2.1. But it can run only tests for net461 and netcoreapp2.1. This is good enough I think.

Reply:
What is the issue with generating 2.0 and 2.1?

Reply:
It causes the test explorer to offer to run tests for 2.0, but that does not work, by lack of an adequate test adapter.